modules/my_health/app/serializers/my_health/v1/prescription_documentation_serializer.rb

```diff
@@ -4,15 +4,13 @@
 
 module MyHealth
   module V1
-    class PrescriptionDocumentationSerializer < ActiveModel::Serializer
-      attributes :html
+    class PrescriptionDocumentationSerializer
+      include JSONAPI::Serializer
 
-      def html
-        object.html[:html]
-      end
+      set_id { '' }
```

Comment:
Can you explain what this is needed for? @stevenjcumming

Reply:
JSON:API specifications require a string for id. So the only way to accommodate a nil `id` is to use a blank string. Active Model Serializer did this out of the box, but `jsonapi-serializers` do not.